src/main/java/org/iac2/service/utility/Edmm.java

```diff
@@ -0,0 +1,75 @@
+package org.iac2.service.utility;
```

Comment:
In theory, in the future, we would move plugins to their own repositories. So the structure will be something like:
- There is an `iac2.common` repository seen by everyone.
- There is an `iac2.<plugin-category>.common` repository for each plugin-type, e.g., `iac2.architecturereconstruction.common`.
- There is an `whatever.whatever` repository per plugin, e.g., `USTUTT.is.the.best.opentoscacontainermodelcreationplugin`.
- There is an `iac2.iacmf` repository, which is the application.

Dependencies are as follows:
- `iac2.common` does not depend on anything else.
- `iac2.<plugin-category>.common` depends on `iac2.common`.
- a concrete plugin of a given category depend on both `iac2.common` and `iac2.<plugin-category>.common`.
- `iac2.iacmf` depends on both `iac2.common` and all repositories `iac2.<plugin-category>.common`, but not on concrete plugins. Plugins are imported at runtime (using some framework, WHICH IS NOT OSGI!!)

Therefore, a utility class like this one should be either placed in `iac2.common` or `iac2.<plugin-category>.common` depending on its nature.

Reply:
fixed